metricbeat/mb/module/wrapper.go

```diff
@@ -102,15 +105,27 @@ func createWrapper(module mb.Module, metricSets []mb.MetricSet, options ...Optio
 		metricSets: make([]*metricSetWrapper, len(metricSets)),
 	}
 
+	//FIXME provide proper struct
+	var hs struct {
+		FailureThreshold int `config:"failureThreshold"`
```

Comment:
might be a good idea to add a validator of [positive](https://github.com/elastic/go-ucfg/blob/587fe67620f33b205484ef84b547d1ad30531035/validator.go#L56), although switching to an unsigned type would make this less important.

Reply:
changed to unsigned in df01e8ea4bd4e255575750a1a42f5a8177dafd44